srcpkgs/php/template

```diff
@@ -219,6 +214,15 @@ php-pear_package() {
 		if [ -n "$CROSS_BUILD" ]; then
 			_env+=" PEAR_PHP=/usr/bin/php"
 		fi
+		# Open bug since July, hacky fix
+		# Bug: https://bugs.php.net/bug.php?id=79869
+		# Proposed fix: https://github.com/pear/PHP_Archive/pull/4
+
+		# This will download the pear installer but ultimately fail because of syntax errors
+		make install-pear ${_env} || true;
+		# Patch the syntax error and remove some spaces to keep the length the same and the zip valid
+		sed -i 's: \$flags \& 0x00010000:\$flags[1]\&0x00010000:' pear/install-pear-nozlib.phar
```

Comment:
Can't we add a distfile? Reading "download" scares me.

And consider using `vsed`.

Reply:
do you think this is better? :)